concepts/05 UI Components/DropDownButton/00 Getting Started with DropDownButton/80 Customize the Drop-Down Menu.md

```diff
@@ -1,4 +1,4 @@
-#include common-customize-dropdown-menu For example, the following code sets the height of the DropDownButton's drop-down menu to 150 pixels:
+The DropDownButton uses the [Popup](/Documentation/Guide/UI_Components/Popup/Getting_Started_with_Popup/) component as the drop-down menu. To customize the menu, specify [Popup properties](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/) in the [dropDownOptions](/Documentation/ApiReference/UI_Components/dxDropDownButton/Configuration/#dropDownOptions) object. For example, the following code sets the height of the DropDownButton's drop-down menu to 150 pixels:
```

Comment:
1. This is unclear to me: we are talking about the only drop-down menu a component can have.
2. ok

Reply:
Yes, the only component, I understand this. Compare with "You use this mechanical manipulator as a right hand." You have only one and only right hand, yet still if you use something instead of it, you refer to a right hand as a general right hand concept. Our example is no different.